src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable, AreaCalculator {
```

Comment:
let's implement these interfaces only in the `Figure` class and remove `implements Drawable, AreaCalculator` from classes that `extends Figure`